src/core/dto/profile.dto.ts

```diff
@@ -0,0 +1,76 @@
+import "reflect-metadata";
+
+import { Length } from "class-validator";
+import { Field, InputType } from "type-graphql";
+
+@InputType()
+export class CreateProfileInput {
+  @Field()
+  @Length(1, 255)
+  fullName!: string;
+
+  @Field()
+  pronouns!: string;
+
+  @Field()
+  tdaGradYear!: number;
+
+  @Field()
+  currentLocation!: string;
+
+  @Field()
+  @Length(1, 500)
+  bio!: string;
+
+  /*
+  @Field(() => String, { nullable: true })
+  profilePic: string | null | undefined;
+
+  @Field(() => String, { nullable: true })
+  homeCountry: string | null | undefined;
+
+  @Field(() => String, { nullable: true })
+  nickname: string | null | undefined;
+
+  @Field(() => String, { nullable: true })
+  website: string | null | undefined;
+
+  @Field()
+  tdaGradYearBannerVisible!: boolean;
+  */
+}
+
+@InputType()
```

Comment:
For the `@Field()` decorators, you can remove the `() => String` and just used the `x: string` on the second line, it is redundant. The only time we will need to put the type on that Field line is if it is a complex or resolved type of our creation, not the primitive types that come with the language.